Source/UIKit/UILabel.swift

```diff
@@ -14,4 +14,8 @@ extension UILabel {
     public var rex_text: MutableProperty<String> {
         return rex_stringProperty("text")
     }
+    
+    public var rex_textColor: MutableProperty<UIColor> {
+        return rex_valueProperty(&textColor, { [weak self] in (self?.textColor)! }, { [weak self] in self?.textColor = $0 })
```

Comment:
How about `self?.textColor ?? UILabel().textColor`? I don't think `self` would be deallocated in normal scenarios so the cost of allocating a new label could be tolerable?

I'm not sure how to create a static `UILabel` in this case (When I tried to add it here Xcode says `Static properties may only be declared on a type`).

I dont like the `textColor` to be an optional since it never is nil. I've just tried this to be sure, but when setting a labels `textColor` to nil, its actually `clearColor` and **not** nil